doc/user/client_service.md

```diff
@@ -1,5 +1,7 @@
-## Client service
-For every etcd cluster created the etcd-operator will create an etcd client service in the same namespace with the name `<cluster-name>-client` .
+# Etcd client service
```

Comment:
Etcd -> etcd. We always use `etcd` even if it is the first word of the sentence.